setup.py

```diff
@@ -41,15 +41,14 @@
 ]
 
 install_requires = [
-    "click>=7",
     "graphviz>=0.12",  # FIXME needed only if yadage visuale=True.
     "networkx==1.11",
     "packtivity==0.14.21",
     "pydot2>=1.0.33",  # FIXME needed only if yadage visuale=True.
     "pydotplus>=2.0.2",  # FIXME needed only if yadage visuale=True.
     "pygraphviz>=1.5",  # FIXME needed only if yadage visuale=True.
     "reana-commons[yadage]>=0.8.0a17,<0.9.0",
-    "requests==2.22.0",
+    "requests>=2.25.1",
```

Comment:
Nice, this fixes the `urllib3` upgrade. I'm not sure why had had such a sharp pin on requests... Must date from very old times. Something to do also for `r-w-controler` and `r-server`...

Reply:
It's a bit strange because we only have this pinned in `r-w-e-yadage`, not in `r-w-e-serial` nor `r-w-e-cwl`. `requests` comes from `bravado` which comes from `reana-commons` so maybe we can remove `requests` from here altogether.